pkg/apis/core/v1beta1/helper/helper.go

```diff
@@ -1247,3 +1250,55 @@ func BackupBucketIsErroneous(bb *gardencorev1beta1.BackupBucket) (bool, string)
 	}
 	return true, lastErr.Description
 }
+
+func ReadImageVector(r io.Reader) (gardencorev1beta1.ImageVector, error) {
+	iv, err := imagevector.Read(r)
+	if err != nil {
+		return nil, err
+	}
+	var images []gardencorev1beta1.ImageSource
+	for _, is := range iv {
+		images = append(images, gardencorev1beta1.ImageSource(*is))
+	}
+	return images, nil
```

Comment:
We don't want to do this right now, isn't this what we agreed? Otherwise we would have to change a lot more code, deal with backward compatibility, etc. Perhaps it makes sense, but let's do it in a different PR. I regard this PR as the minimal change required to introduce strongly typed image vectors and use them in the API. My goal was to touch the existing `imagevector` as little as possible, and do it in a compatible way.

Under these assumptions, concretely the method above has to stay as it is, since we are constrained by the representation already used in existing configuration files / configmaps.

Reply:
Ok for me. Let's do it later on.